server/services/zigbee2mqtt/utils/convertValue.js

```diff
@@ -14,11 +14,26 @@ function convertValue(feature, state) {
       result = state === 'ON' ? 1 : 0;
       break;
     }
-    default:
-      if (typeof state === 'string') {
-        throw new Error(`Zigbee2mqqt don't handle value "${state}" for feature "${feature}".`);
-      }
+    // Case for Button devices
+    case 'click': {
       result = state;
```

Comment:
What is the `state` type in this case ? A string ? A number ?

Reply:
state represents a string corresponding to state given by Zigbee2mqtt.
Examples for tested button devices:
Xiaomi Button states => single, double, triple or quadruple
IKEA 0-1 Button states => on, off, brightless_up, brightless_down et brightless_stop
state is directly passed to Gladys and can be seen on the dashboard

Reply:
So, why  you don't add `typeof string` case ? As the number or bool.

Reply:
So the `click` is not required anymore, isn't it ?